### PR TITLE
Feature/created instructor booking page and instructor available courses

### PR DESCRIPTION
**Pull request by Jie-chelchel**
What I did:

Created a route for the Instructor Booking page
Imported Banner into Instructor booking page
created a InstructorAvailableCourses.js file

Files changed:
src/index.js
InstructorBooking/index.js

Files added:
InstructorBooking/InstructorAvailableCourses.js

Why were these changes made?
To create instructor booking page, and a file that will later render instructor course card.

!(https://www.loom.com/share/1da9b7851b3a4fefad8cb542812d1d84)

### CODE REVIEW

File: src/styles/InstructorStyles/index.less [hunk 1]
```diff
@@ -39,15 +39,13 @@
 }
 
 .dashboard-banner {
-  background-color: #006c72; //  orange colorFFAD00
```
Comment (Rodgers31) [inline src/styles/InstructorStyles/index.less:42]: Commented-out styles if not needed should probably be removed before merging with main.

**Review by Rodgers31 — APPROVED** [review]
Apart from the commented-out styles and conflict which looks to be just spacing. Everything else looks good.

**Review by muhannadbm — APPROVED** [review]
Everything looks good, though commented code should be removed

**Review by ashtilawat23 — APPROVED** [review]
Qualified

**Pull request merged**